src/lib_ccx/params.c

```diff
@@ -1114,6 +1113,15 @@ int parse_parameters (struct ccx_s_options *opt, int argc, char *argv[])
 			version(argv[0]);
 			return EXIT_WITH_HELP;
 		}
+
+		if(!strcmp(argv[i], "-man"))	//Generate manual page
+		{
+#ifdef __linux__
+			system("bash manscript.sh");
+			mprint("Finished executing man script\n");
+#endif
+			return EXIT_WITH_HELP;
+		}
```

Comment:
This is a really really weird solution - if ccextractor (the binary) cannot generate the man page directly, then the correct way to generate it would be for manscript.sh to call ccextractor -help to get the output, and then do the transformation from the script, generate the man page, and possibly install it (if a install parameter is present).

Otherwise you're forcing the user to have manscript.sh around. Where would be this file be? ccextractor usually goes in /usr/local/bin or /usr/bin, but a file called manscript.sh doesn't really go there...

Reply:
I agree with this. It seems a weird solution, and it makes quite a lot of assumptions.